src/main/java/org/jabref/gui/actions/CopyFilesResultViewModel.java

```diff
@@ -0,0 +1,50 @@
+package org.jabref.gui.actions;
+
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIcon;
+
+public class CopyFilesResultViewModel {
+
+    private final StringProperty file = new SimpleStringProperty("");
+    private final ObjectProperty<MaterialDesignIcon> icon = new SimpleObjectProperty<>(MaterialDesignIcon.ALERT);
+
+    private final StringProperty message = new SimpleStringProperty("");
+
+    public CopyFilesResultViewModel(String file, boolean success, String message) {
```

Comment:
maybe change the name `Result` -> `ResultItem` to make it clearer that is one item in the result window.